docs/_docs/windows.md

```diff
@@ -42,16 +43,24 @@ Now all that is left to do is install Jekyll.
 sudo gem install jekyll bundler
 ```
 
-You can test by running:
+Check to see if your installation worked.
```

Comment:
Prefer rewording to `To check if Jekyll installed properly, run the following:`
But I'll leave it to your judgement.